src/steps/destination/converter.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  createDirectRelationship,
+  createIntegrationEntity,
+  Entity,
+  parseTimePropertyValue,
+  Relationship,
+  RelationshipClass,
+} from '@jupiterone/integration-sdk-core';
+import { Destination, DestinationList } from '../../types';
+
+import { Entities } from '../constants';
+
+export function createDestinationListKey(id: number) {
+  return `${Entities.DESTINATION_LIST._type}:${id}`;
+}
+
+export function createDestinationKey(id: string) {
+  return `${Entities.DESTINATION._type}:${id}`;
+}
+
+export function createDestinationListEntity(
+  destinationList: DestinationList,
+): Entity {
+  return createIntegrationEntity({
+    entityData: {
+      source: destinationList,
+      assign: {
+        _key: createDestinationListKey(destinationList.id),
+        _type: Entities.DESTINATION_LIST._type,
+        _class: Entities.DESTINATION_LIST._class,
+        name: destinationList.name,
+        displayName: destinationList.name,
+        access: destinationList.access,
+        isGlobal: destinationList.isGlobal,
+        createdOn: parseTimePropertyValue(destinationList.createdAt, 'sec'),
+        modifiedOn: parseTimePropertyValue(destinationList.modifiedAt, 'sec'),
+        isMspDefault: destinationList.isMspDefault,
+        isMarkedForDeletion: destinationList.markedForDeletion,
```

Comment:
```suggestion
        mspDefault: destinationList.isMspDefault,
        markedForDeletion: destinationList.markedForDeletion,
```
This one I could go either way on, up to you.